optuna/trial.py

```diff
@@ -372,3 +445,116 @@ def system_attrs(self):
         """
 
         return self.storage.get_trial_system_attrs(self.trial_id)
+
+
+class FixedTrial(BaseTrial):
+
+    """A trial class which suggests a fixed value for each parameter.
+
+    This object has the same methods as :class:`~optuna.trial.Trial`, and it suggests pre-defined
+    parameter values. The parameter values can be determined at the construction of the
+    :class:`~optuna.trial.FixedTrial` object. In contrast to :class:`~optuna.trial.Trial`,
+    :class:`~optuna.trial.FixedTrial` does not depend on :class:`~optuna.study.Study`, and it is
+    useful to deploy optimization results.
+
+    Example:
+
+        Evaluate an objective function with parameter values given by a user:
+
+        .. code::
+
+            >>> def objective(trial):
+            >>>     x = trial.suggest_uniform('x', -100, 100)
+            >>>     y = trial.suggest_categorical('y', (-1, 0, 1))
```

Comment:
This example works, but using a list (i.e., `[-1, 0, 1]`) as the value of `choices` seems to be preferable because it is consistent with other examples.

Reply:
Hmm. Actually, I copied this line from [`examples/quadratic_simply.py`](https://github.com/pfnet/optuna/blob/master/examples/quadratic_simple.py#L27).  Do you think we should fix both?

Reply:
[MEMO] After a brief conversation, we came to the conclusion that we use lists for that purpose in this repository ( `quadratic_simple.py` will be fixed in another PR).